OmiseSwiftTests/ChargesOperationFixtureTests.swift

```diff
@@ -1137,7 +1137,54 @@ class ChargesOperationFixtureTests: FixtureTestCase {
         XCTAssertEqual(items[4].name, "metadata[customer_id]")
         XCTAssertEqual(items[4].value, "123")
     }
+
+    func testMobileBankingChargeCreate() {
+        let expectation = self.expectation(description: "Mobile Banking SCB Charge create")
+        
+        let createParams = ChargeParams(value: Value(amount: 100_00, currency: .thb),
+                                        sourceType: .mobileBanking(.scb),
+                                        returnURL: defaultReturnURL)
+        
+        Charge.create(using: testClient, params: createParams) { (result) in
+            defer { expectation.fulfill() }
+            
+            switch result {
+            case let .success(charge):
+                XCTAssertNotNil(charge)
+                XCTAssertEqual(charge.value.amount, 10_000_00)
+                XCTAssertEqual(charge.source?.paymentInformation, .mobileBanking(.scb))
+                XCTAssertEqual(charge.returnURL, defaultReturnURL)
+            case let .failure(error):
+                XCTFail("\(error)")
+            }
+        }
+        
+        waitForExpectations(timeout: 15.0, handler: nil)
+    }
     
+    func testMobileBankingChargeRetrieve() {
```

Comment:
How about to put a more specific `Bank` on the testing method name like `testSCBMobileBankingChargeRetrieve` follow the other test cases.